src/Aquifer.Jobs/Services/TranslationProcessors/FrenchTranslationProcessingService.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.Text.RegularExpressions;
+using Aquifer.Common.Utilities;
+
+namespace Aquifer.Jobs.Services.TranslationProcessors;
+
+public sealed partial class FrenchTranslationProcessingService : ILanguageSpecificTranslationProcessingService
+{
+    private const char NarrowNonBreakingSpace = '\u202f';
+    private const char NonBreakingSpace = '\u00a0';
+    private const string HtmlEncodedNonBreakingSpace = "&nbsp;";
+
+    private static readonly IReadOnlySet<string> s_RegexItemsThatHaveNnbspAfterSelf = new HashSet<string>
+    {
+        @"\bv\.", // v.
+        @"\bp\.", // p.
+        "«",
+        "=",
+        @"\bapr\.", // apr.
+        @"\bav\." // av.
+    };
+
+    private static readonly IReadOnlySet<string> s_RegexItemsThatHaveNnbspBeforeSelf = new HashSet<string>
+    {
+        "»",
+        "!",
+        @"\?",
+        ":",
+        ";"
+    };
+
+    private readonly string _nnbspAfterPattern = @$"({string.Join("|", s_RegexItemsThatHaveNnbspAfterSelf)})(?=[A-Za-zà-üÀ-Ü0-9\x20])";
```

Comment:
Can a regular space be used instead of `\x20`?

Reply:
Oh, for clarity. I think it's easy to miss a trailing space like that and think it's unintentional. This way it's explicit what's intended.